android/build.gradle

```diff
@@ -57,7 +57,6 @@ dependencies {
     testImplementation 'junit:junit:4.13.2'
     androidTestImplementation 'androidx.test.ext:junit:1.1.3'
     androidTestImplementation 'androidx.test.espresso:espresso-core:3.4.0'
-    implementation("com.swmansion.starknet:starknet:0.0.1")
```

Comment:
Why was this removed?

Reply:
Because it was a mistake. Android is the lib for android ecosystem. This should be added in androiddemo package